client/README.rst

```diff
@@ -1,4 +1,6 @@
 .. _beadledom-client:
 
+Deprecated since 4.0, use `Retrofit <https://github.com/square/retrofit>`_ instead.
```

Comment:
Do we want to deprecate this in a 3.x release even?

Reply:
Would be then be removing it from the 4.0 release?

Reply:
I wish we could do that, but we probably have too many consumers to remove it in 4.0, so it would just be deprecated in both. Maybe re-evaluate removal in 5.0. Thoughts, @nathanschile?

Reply:
I agree with @johnlcox We want people to upgrade to 4.0, so lets leave it in.